docs/standard/garbage-collection/fundamentals.md

```diff
@@ -116,7 +116,7 @@ Garbage collection primarily occurs with the reclamation of short-lived objects.
 
 - **Generation 0**. This is the youngest generation and contains short-lived objects. An example of a short-lived object is a temporary variable. Garbage collection occurs most frequently in this generation.
 
-  Newly allocated objects form a new generation of objects and are implicitly generation 0 collections. However, if they are large objects, they go on the large object heap in a generation 2 collection.
+  Newly allocated objects form a new generation of objects and are implicitly generation 0 collections. However, if they are large objects, they go on the large object heap (LOH) in a generation 2 collection. (The LOH is sometimes referred to as *generation 3*. It's a physical generation that's logically part of generation 2.)
```

Comment:
the addition looks fine but I find the original text confusing 'cause it sounds like large objects start in gen0 and only are considered as part of LOH when a gen2 GC happens which is not the case - probably better to say it like this:

`Newly allocated objects form a new generation of objects and are implicitly generation 0 collections. However, if they are large objects, they go on the large object heap (LOH) which is sometimes referred to as *generation 3*. It's a physical generation that's logically collected as part of generation 2.`